### ISSUE TITLE
Relax: Brush flickers during navigation

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#1377]
If you are using Relax and hover your mouse over a surface so that the brush circle is displayed, rotating, panning, or zooming the view from that point causes the brush to flicker. The easiest fix would probably be to just not display the brush during navigation, since it can't be used then anyway.

**Comment by vxlcoder** [comment]
this does not seem to happen on my machine.  are you able to capture this in a video screenshot?

**Comment by jlampel** [comment]
Oh, weird! Sure thing:

https://github.com/user-attachments/assets/cf256905-e429-42c2-85ad-5c1b1f3aed50

**Comment by vxlcoder** [comment]
ah!  are you using a trackpad??

**Comment by jlampel** [comment]
Nope! Regular three button mouse on Windows 11. I am using Emulate 3
Button mouse in Blender's preferences, but beyond that it's pretty
standard. 

On December 20, 2024, GitHub ***@***.***> wrote:
> ah! are you using a trackpad??
>

—
Reply to this email directly, view it on GitHub
<https://github.com/CGCookie/retopoflow/issues/1377#issuecomment-
2557510202>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/ADDBQHFPVVUVOYOKV7PKVID2GROBPAVCNFSM6AAAAABT346US2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNJXGUYTAMRQGI>.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by vxlcoder** [comment]
@jlampel, can you give 602dfaa a test when you get a chance?

I'm not able to reproduce this bug on my machines.

**Comment by jlampel** [comment]
Confirmed fixed!
